delta/sourcing-psql/src/test/scala/ch/epfl/bluebrain/nexus/delta/sourcing/stream/pipes/SelectPredicatesSuite.scala

```diff
@@ -76,4 +76,11 @@ class SelectPredicatesSuite extends BioSuite {
     pipe(Set(rdfs.label)).apply(element).assert(expected)
     defaultPipe.apply(element).assert(expected)
   }
+
+  test("Do not apply any modifications for a forward type 'File'") {
+    val types = Set(nxv + "PullRequest", nxv + "File")
+    val file  = element.copy(value = element.value.copy(types = types))
+    pipe(Set(rdfs.label)).apply(element).assert(file)
+    defaultPipe.apply(file).assert(file)
```

Comment:
Why is element passed to the first, but file to the second?

Reply:
It is still an elem which contains a file. I will make that more explicit